src/python/WMCore/WorkQueue/Policy/Start/Block.py

```diff
@@ -45,15 +45,22 @@ def split(self):
                         parentList[dbsBlock["Name"]] = self.sites
                     else:
                         parentList[dbsBlock["Name"]] = self.siteDB.PNNstoPSNs(dbsBlock['PhEDExNodeList'])
-
+            
+            if self.args['SliceType'] == 'NumberOfEvents' and block['NumberOfEvents'] == 0:
+                # if there is 0 events file and use EventAwareLumiBased algo
+                # set the estimate jobs to 1 (It is hard to get the realistic estimate 
+                # unless run the splitting alogrithm
+                estimateJobs = 1
+            else:
+                estimateJobs = ceil(float(block[self.args['SliceType']]) / float(self.args['SliceSize'])),
```

Comment:
I'd suggest to just use this formula above in a `max(1, formula)`. For clarity, a line of comment saying that there may be 0 event files.